app/style/variables.js

```diff
@@ -18,3 +24,9 @@ export const notificationColors = {
     color: 'black'
   }
 }
+
+export const StyledText = styled.Text`
```

Comment:
Let's put this in a file in `components/` and then use it in place of the various `Text` components created across the app.

Reply:
I've added the `react-native-global-props` package to set the style across the whole app